src/components/ux-alert/ux-alert.hbs

```diff
@@ -0,0 +1,8 @@
+<div data-alert class="alert-box {{class}}">
+	{{#if isDataModel}}
+		{{{text}}}
+	{{else}}
+		{{yield}}
+	{{/if}}
+	<a href="#" class="close">&times;</a>
```

Comment:
Can't see this close handled anywhere? It should replicate how it works on the docs:
http://foundation.zurb.com/docs/components/alert_boxes.html